### PR TITLE
update readme and add contributing file, fixes #49

### LINKED ISSUE #49: Update contributing in readme
The instruction for rebase (step 4) should be `git pull --rebase git@github.com:koombea/kiliframework.git develop` if rebasing against original repository or `git pull --rebase develop` if rebasing against forked repository

### PR DESCRIPTION
**Pull request by rhonaldomaster**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# How to contribute
+
+We want the best for our framework, and contributing to it is one of the best ways to do it.
```
Comment (dbohorquez8) [inline CONTRIBUTING.md]: Yo removería esta frase. No tiene mucho sentido

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Kili Framework is open source, and we love to read your ideas about improvements for making it even more awesome.
```
Comment (dbohorquez8) [inline CONTRIBUTING.md]: "we'd love"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+If you want to contribute, please ensure you follow the guidelines below :)
+
+## Contributing to core
```
Comment (dbohorquez8) [inline CONTRIBUTING.md]: Core. (Mayúsculas)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+### Reporting Issues
+
+This is a great way to became a contributor as it doesn't require technical skills. If you spot a bug, or want to help to solve one, you can go to [issues](https://github.com/koombea/kiliframework/issues/) and see what issues are open or create a new one.
+
+### Making Changes
+
+1. Fork the project (<https://github.com/koombea/kiliframework>).
+    1. If you already forked the project, rebase your develop branch (`git pull --rebase git@github.com:koombea/kiliframework.git develop`)
+    2. Ensure you follow the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/coding-standards/php/)
+    3. Optionally it's possible install [EditorConfig](http://editorconfig.org/) on your editor to auto configure with indentation, line endings and other settings
```
Comment (dbohorquez8) [inline CONTRIBUTING.md]: It is recommended to install....

**Comment by rhonaldomaster** [comment]
@dbohorquez8 i updated the request with the comments you did

**Pull request merged**